### DISCUSSION TITLE
Intervals live data

### DISCUSSION THREAD

**Discussion by SoulFliktioN** [br-g/openf1#28] [Q&A] [answered]
Hey! 
First let me say thank you, what an awesome API! I tried to access the intervals today but didn't get any data back. Was there no data or did I do something wrong? All the other data was fine.

**Comment by br-g** [comment]
Hello! Thank you for your kind words.
It seems that the interval data was not released by the official F1 data source.
I will look into that!

**Reply by PedreAltair** [reply]
Sorry to bother you but I've noticed this issue persists in other sessions. In my case I've made some test with session 9161 and it returns an empty json.

**Reply by br-g** [reply]
Fixed!

**Comment by br-g** [comment, marked as answer]
There seems to have been some change in the incoming data.
Should be fixed now!

**Reply by PedreAltair** [reply]
Can confirm. Thank you very much for the speedy fix

**Reply by SoulFliktioN** [reply]
No worries please! I mean you are doing the work. Sometimes it is just hard to figure out where the problem is.

**Reply by br-g** [reply]
Sorry intervals seem to be broken again. I will have a look at this

**Reply by michae1michae1** [reply]
Hello all!

Not exactly a solution to this problem, but when I noticed this I had an idea... I am working on a way to get interval and gap to leader by using position data.

In theory, it will result in a higher resolution of intervals data because the position data updates more frequently

The idea is that we can check the position of the car, every tick, and through some math... we can find the amount of progress through the lap at each tick - with this, we can compare the timestamps and generate an interval and gap for each driver

You have to check the distance between each location tick, and then add all of the distances and convert each tick to a percentage of track complete

anyway - I won't have much time this week to complete this, but thought I would post this here and maybe someone will get it done before me (:

**Reply by br-g** [reply]
Fixed!
